wp-content/civi-extensions/goonjcustom/Civi/InductionService.php

```diff
@@ -173,11 +173,13 @@ private static function findCoordinatorForOffice($officeId) {
    * Common logic to send an email.
    */
   private static function sendInductionEmail($contactId) {
```

Comment:
can we say `volunteerId` instead of `contactId`?

Reply:
SURE